### PR TITLE
Basic ui

### PR DESCRIPTION
**Pull request by taltaub22**

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Client
```
Comment (idanstark42) [inline client/README.md]: Remove this file

File: index.js [hunk 2]
```diff
@@ -23,7 +23,13 @@ app.use(bodyParser.json())
 app.use(msCorrelation.correlationMiddleware)
 
 const tournamentDataRouter = require('./routers/tournamentDataRouter')
-const matchTeamRouter = require('./routers/matchTeamRouter');
+const matchTeamRouter = require('./routers/matchTeamRouter')
+
+app.use((req, res, next) => {
```
Comment (idanstark42) [inline index.js]: This is the exact same change we saw in another PR... See the comments there, You should use the already ready pacakge [cors](https://www.npmjs.com/package/cors)

**Comment by taltaub22** [comment]
The ui is moving to another repository

**Comment by idanstark42** [comment]
Isn't this coming back here?

**Pull request closed without merging**